app/bingey-api/models/Users.js

```diff
@@ -0,0 +1,11 @@
+const { Schema, model } = require('mongoose');
+
+const UsersSchema = new Schema({
+    username: String,
+    password: String,
+    watchlistLink: String,
+});
+
+const Users = model('Users', UsersSchema);
```

Comment:
The model name should be singular, as it will be pluralized when establishing a connection with the collection